src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1639,25 +1642,25 @@ def validate(self) -> None:
         if (self.max_namespaces_with_netmask and self.max_namespaces_with_netmask < 0):
             raise SpecValidationError("Max namespaces with netmask can't be negative")
 
-        if type(self.max_subsystems) != int:
+        if not isinstance(self.max_subsystems, int):
```

Comment:
I'd suggest that you split this and three similar fixups below into a separate PR to expedite addressing `make check` failures.